nix/cells/marlowe-playground/scripts/assemble-changelog.nix

```
@@ -0,0 +1,27 @@
+{ inputs, cell }:
+
+let
+  inherit (cell.library) writeShellScriptInRepoRoot purescript;
+  inherit (purescript) spago2nix;
```

Some not used stuff here

```suggestion
  inherit (cell.library) writeShellScriptInRepoRoot;
```